theme_kea/views/snippets/s_picture.xml

```diff
@@ -9,7 +9,7 @@
     </xpath>
     <!-- Shape -->
     <xpath expr="//div[hasclass('container')]" position="before">
-        <div class="o_we_shape o_web_editor_Wavy_03 o_we_flip_y"/>
```

Comment:
> The picture in this snippet seems broken?

Fortunately only on master...
It is because of this  https://github.com/odoo/odoo/commit/934069f834534ca3553ed28e46e5619c33463e50
A `design-themes` branch is missing to adapt the image shapes, the image shapes folders/categories are no longer the same and the theme templates using image shapes need to be adapted. Maybe we can ask mano-odoo to do it, or I could do it.. whatever you prefer..

Reply:
I'll ask mano-odoo, thanks :+1:

Reply:
@mano-odoo We missed the fact a design-themes PR was needed with https://github.com/odoo/odoo/pull/110673 since some shapes' paths have changed :grimacing: Could you create a PR to adapt the repo as it should? For instance: theme_kea -> picture snippet -> broken. Thanks :)

cc @rdeodoo

Reply:
Indeed, my bad to not notice it during the review :facepalm:
Let us know as soon as you have a fix :+1:

Reply:
https://github.com/odoo/design-themes/pull/641 should do 👍
lmk if you need anything 😄